test/testvm.py

```diff
@@ -679,15 +750,18 @@ def _start_qemu(self, maintain=False, tty=False, monitor=None):
         cmd = [
             self._locate_qemu_kvm(),
             "-m", str(MEMORY_MB),
-            "-drive", "if=virtio,file=%s,index=0,serial=ROOT,snapshot=%s" % (self._image_root, snapshot),
-            "-kernel", self._image_kernel,
-            "-initrd", self._image_initrd,
-            "-append", "root=/dev/vda console=ttyS0 quiet %s" % (selinux, ),
+            "-drive", "if=virtio,file=%s,index=0,serial=ROOT,snapshot=%s" % (drive_to_start, snapshot),
             "-nographic",
```

Comment:
> Let's make it so that ./vm-run drops "-nographic". That gives us a working console again.

We can't assume that everyone will have graphic output. How about `-display none`?